packages/frontend/src/ee/features/aiCopilot/components/AgentDetails.tsx

```diff
@@ -74,6 +75,11 @@ export const AgentDetails: FC = () => {
             enabled: !!agentUuid,
         },
     );
+    const { mutateAsync: deleteAgent } = useDeleteAiAgentMutation({
+        onSuccess: () => {
+            void navigate('/generalSettings/aiAgents');
+        },
+    });
```

Comment:
do we need this `onSuccess` here? We should probably `deleteAgent().then(navigate...)` instead.

Also we need to invalidateQuery because the deleted agent appears in the list after deletion